src/language.rs

```diff
@@ -66,16 +66,14 @@ impl Language {
                 }
             }
 
-            return Err(FormatterError::Query(
-                format!("Cannot detect language from unknown extension: '{filename}'"),
-                None,
-            ));
+            return Err(FormatterError::LanguageDetection(format!(
```

Comment:
Actually now that I think about it, it would be nice if `FormatterError::LanguageDetection` took just the file name (and possibly extension) as parameter, and left it to `FormatterError::fmt` to take care of the error message.

Reply:
Good idea :ok_hand:

Reply:
Done in 6d439aabc376df6